Build/CMake/Modules/AtomicCommon.cmake

```diff
@@ -1,5 +1,5 @@
 #
-# Portions Copyright (c) 2008-2016 the Urho3D project.
+# Portions Copyright (c) 2008-2017 the Atomic project.
 #
```

Comment:
In this specific case i changed copyright because `AtomicCommon.cmake` diverged from original file so much that syncing up would not be viable nor would it be a good idea since Urho's build system is upside-down and now we have that part fixed. So maybe this is ok after all? For this very specific reason i did not format any scripts in ThirdParty.